faunadb-java-dsl/src/main/java/com/faunadb/client/query/Pagination.java

```diff
@@ -141,7 +138,8 @@ public Pagination size(Integer size) {
   /**
    * Define if the pagination should include sources of not
```

Comment:
I believe "source of not" should be source or not.   Maybe something like this.

Define if the pagination should return source information or not, by default source information is not returned